mp/src/materialsystem/stdshaders/momentum_dx9_30.txt

```diff
@@ -1,30 +1,27 @@
-pbr_ps30.fxc
-pbr_vs30.fxc
+gauss_blur_3x3_ps30.fxc
+gauss_blur_3x3_vs30.fxc
 
-//gauss_blur_3x3_ps30.fxc
-//gauss_blur_3x3_vs30.fxc
-//
-//SDK_skin_ps20b.fxc
-//SDK_skin_vs20.fxc
-////SDK_vertexlit_and_unlit_generic_bump_ps20b.fxc
-//SDK_vertexlit_and_unlit_generic_bump_ps2x.fxc
-//SDK_vertexlit_and_unlit_generic_bump_vs20.fxc
-////SDK_vertexlit_and_unlit_generic_ps20b.fxc
-//SDK_vertexlit_and_unlit_generic_ps2x.fxc
-//SDK_vertexlit_and_unlit_generic_vs20.fxc
-//
-//SDK_cloak_blended_pass_ps2x.fxc
-//SDK_cloak_blended_pass_vs20.fxc
-//SDK_emissive_scroll_blended_pass_ps2x.fxc
-//SDK_emissive_scroll_blended_pass_vs20.fxc
-//
-//SDK_flashlight_ps2x.fxc
-//SDK_lightmappedgeneric_flashlight_vs20.fxc
-////SDK_lightmappedgeneric_ps20b.fxc
-//SDK_lightmappedgeneric_ps2x.fxc
-////SDK_lightmappedgeneric_filter0_ps30.fxc
-////SDK_lightmappedgeneric_filter1_ps30.fxc
-////SDK_lightmappedgeneric_filter2_ps30.fxc
-//SDK_lightmappedgeneric_vs20.fxc
-//
-////decalmodulate_ps2x.fxc
+SDK_skin_ps20b.fxc
+SDK_skin_vs20.fxc
+//SDK_vertexlit_and_unlit_generic_bump_ps20b.fxc
+SDK_vertexlit_and_unlit_generic_bump_ps2x.fxc
+SDK_vertexlit_and_unlit_generic_bump_vs20.fxc
+//SDK_vertexlit_and_unlit_generic_ps20b.fxc
+SDK_vertexlit_and_unlit_generic_ps2x.fxc
+SDK_vertexlit_and_unlit_generic_vs20.fxc
+
+SDK_cloak_blended_pass_ps2x.fxc
+SDK_cloak_blended_pass_vs20.fxc
+SDK_emissive_scroll_blended_pass_ps2x.fxc
+SDK_emissive_scroll_blended_pass_vs20.fxc
+
+SDK_flashlight_ps2x.fxc
+SDK_lightmappedgeneric_flashlight_vs20.fxc
+//SDK_lightmappedgeneric_ps20b.fxc
+SDK_lightmappedgeneric_ps2x.fxc
+//SDK_lightmappedgeneric_filter0_ps30.fxc
+//SDK_lightmappedgeneric_filter1_ps30.fxc
+//SDK_lightmappedgeneric_filter2_ps30.fxc
+SDK_lightmappedgeneric_vs20.fxc
+
+//decalmodulate_ps2x.fxc
```

Comment:
Ah, you ran into this bug. This is actually a really bizarre artifact of the repo that we aren't really sure how to prevent. But you're going to want to reset back and follow another type of fix:

1. Do the things I suggested above, but do not commit it.
2. Instead, do `git reset HEAD~3` to bring your branch back 3 commits ago. This will bring all your changes back for staging and committing.
3. Re-create your initial feature commit as you had it, **DO NOT COMMIT THE `momentum_dx9_30.txt` AND `momentum_dx9_20b.txt` FILES!**
4. With your actual feature work in a commit (**AND NOT PUSHED**), and these shader files in staging (**NOT COMMITTED**), you can now do the following to fix this weird bug:
```
git rm .gitattributes
git add -A
```
And discard the change to the gitattributes file.

5. These two files should be gone. If so, you can now `git push --force` to update this branch, cleaning it from these files (as well as the extra commits you did before!)

Any doubts about this, just PM me.